common/scripts/copyReferenceFilesToStaging.js

```diff
@@ -0,0 +1,69 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/*
+1. Creates a staging-directory for the docs build
+2. Copies the folders from itwinjs-core/docs and itwinjs-core/generated-docs.
+3. Provides the ability for users to give an absolute path
+4. Provides the ability to modify the default path (itwinjs-core/staging-docs) through CLI arguments
+Usage: node copyReferenceFilesToStaging.js staging-docs-directory.
+*/
+const path = require('path');
+const process = require('process');
+const childProcess = require("child_process");
+const options = {
+  encoding: "utf8"
+};
+
+let fse;
+try {
+  fse = require("fs-extra");
+} catch (err) {
+  console.log("Please check that fs-extra is installed");
```

Comment:
wouldnt it be better to not swallow the error and log it? if its missing the dep it just throw an error anyway

Reply:
We thought it might be more clear to the user. Could throw.

Reply:
i defer to you guys, this might be too nit picky. feel free to resolve both comments

Reply:
Nah definitely good to consider.